Eggstravaganza/Assets/Scripts/ItemsSpawn.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using UnityEditor;
```

Comment:
nit: could you wrap this and the corresponding CustomEditor in an `if UNITY_EDITOR` conditional to prevent build errors?

Reply:
yes this will b important bc i'm making a bunch of builds all the time for multiplay

Reply:
yes good catch